Add Hebart2023 Dataset

linus-md opened the pull request:
This PR includes the packaging scripts for the THINGS data and adds the necessary values to ``lookup.csv``.

mschrimpf commented on packaging/hebart2023/stimulus_set.py:
```diff
@@ -0,0 +1,44 @@
+for index, row in df.iterrows():
+    image_id = row.iloc[0] 
+    image_paths[image_id] = stimulus_path + 'reference_images/' + row.iloc[1][1:]
+    stimuli.append({
```
this is fine, but just to make sure: is there any more meta information we can include about the stimuli? eg category, positions, etc? We generally try to include as much information as possible.

linus-md replied:
I could include wordnet id and synset see [here](https://osf.io/sz2ux). Do you think this would be useful? Because of different number of sunsets the structure of the DataFrame might become annoying.

linus-md replied:
There is also [this](https://osf.io/xtafs) and [this](https://osf.io/65dku).

mschrimpf replied:
generally I would say whatever metadata you can add without a huge hassle, add 👍

mschrimpf reviewed (commented):
Looks good! The only thing I would add are some unit tests to check e.g. that it's the right number of stimuli. See here for instance: https://github.com/brain-score/brain-score/blob/master/tests/test_benchmarks/test_igustibagus2024.py

linus-md commented:
The tests should be passing now. I changed the DataAssembly and StimulusSet such that the ``stimulus_id``is the image number in both. I also updated the lookup.csv with the added rows from the file in my virtual environment.

But there is apparently a problem with the hash:
```=============================================================== short test summary info ================================================================
ERROR test_hebart2023.py - OSError: File '/Users/linussommer/.brainio/assy_Hebart2023/assy_Hebart2023.nc': invalid SHA-1 hash 90f66d37c202dcd1c74d82854efd4a1e8c5fe82e (expect...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================ 24 warnings, 1 error in 0.96s =============================================================```

linus-md commented:
To reproduce the error:
```
import brainscore
assembly = brainscore.get_assembly(name='Hebart2023')
stimulus_set = assembly.attrs['stimulus_set']
```
causes this
```
Loading catalog from entrypoints
Loading lookup from /Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainscore/lookup.csv
Traceback (most recent call last):
  File "/Users/linussommer/Documents/GitHub/brain-score/tests/test_benchmarks/test_temp.py", line 2, in <module>
    assembly = brainscore.get_assembly(name='Hebart2023')
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainscore/__init__.py", line 15, in get_assembly
    assembly = brainio_get_assembly(name)
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainio/fetch.py", line 168, in get_assembly
    assembly = loader.load()
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainio/assemblies.py", line 499, in load
    result = super(StimulusMergeAssemblyLoader, self).load()
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainio/assemblies.py", line 487, in load
    result = super(StimulusReferenceAssemblyLoader, self).load()
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/brainio/assemblies.py", line 459, in load
    result = xr.open_dataarray(self.file_path, group=self.group, chunks=chunks)
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/xarray/backends/api.py", line 666, in open_dataarray
    **kwargs,
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/xarray/backends/api.py", line 479, in open_dataset
    engine = plugins.guess_engine(filename_or_obj)
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/xarray/backends/plugins.py", line 110, in guess_engine
    engines = list_engines()
  File "/Users/linussommer/miniforge3/envs/brainscore/lib/python3.7/site-packages/xarray/backends/plugins.py", line 105, in list_engines
    entrypoints = entry_points().get("xarray.backends", ())
AttributeError: 'EntryPoints' object has no attribute 'get'
```

linus-md commented:
Apparently 
```
import brainscore
stimulus_set = brainscore.get_stimulus_set(name='Hebart2023') 
```
is working.
